src/Resources/Resources/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 6.13.1
+* Fix ABAC condition not included in role definition payload. Get-AzRoleDefinition will now contain ABAC Condition and ConditionVersion info when applicable.
```

Comment:
```suggestion
* Fixed ABAC condition not included in role definition payload. `Get-AzRoleDefinition` will now contain ABAC Condition and ConditionVersion info when applicable.
```

Reply:
Do need to operate the versioning and keep the changelog just under the title `## Upcoming Release`